### PR TITLE
ingredients-analysis

### PR DESCRIPTION
**Pull request by teolemon**
ingredients-analysis

### CODE REVIEW

File: lang/en/texts/ingredients-analysis.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+<p>The egg symbolizes the vegetarian diet, which allows animal derived products like milk, eggs and honey.</p>
+
+<p>The leaf symbolizes the vegan diet which only allows ingredients from vegetal sources.</p>
+
+<p>The monkey symbolizes the palm oil status, since it has been largely impacted by palm-oil related deforestation.</p>
+
+How it works
+
+<p>The detection is based on the ingredient analyzer of Open Food Facts.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/ingredients-analysis.html:24]: On the Google doc, I proposed a new wording for this paragraph and the next.

File: lang/en/texts/ingredients-analysis.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+<p>Last summer, we overhauled it to be able to assess the degree of food processing. Over the past year, it has become more multilingual and is able to deal with even more corner cases, synonyms and subtleties in wording. Based on this work, we were able to add vegan, vegetarian diets as well as a revamped palm oil detector, something we pioneered in 2012.</p>
+
+<p>The identification of vegan, vegetarian and palm oil free products is based on the ingredient analyzer of Open Food Facts. For each product, we analyze the ingredients list to separate each individual ingredient and look for the corresponding ingredient in our ingredients taxonomy.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/ingredients-analysis.html:28]: Starting from here, I think it is the paragraph I wrote to replace the ones above I believe.

File: lang/en/texts/ingredients-analysis.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+<p>The leaf symbolizes the vegan diet which only allows ingredients from vegetal sources.</p>
+
+<p>The monkey symbolizes the palm oil status, since it has been largely impacted by palm-oil related deforestation.</p>
+
+How it works
+
+<p>The detection is based on the ingredient analyzer of Open Food Facts.</p>
+
+<p>Last summer, we overhauled it to be able to assess the degree of food processing. Over the past year, it has become more multilingual and is able to deal with even more corner cases, synonyms and subtleties in wording. Based on this work, we were able to add vegan, vegetarian diets as well as a revamped palm oil detector, something we pioneered in 2012.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/ingredients-analysis.html:26]: This paragraph has a new proposed wording below.

File: lang/en/texts/ingredients-analysis.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+<p>The identification of vegan, vegetarian and palm oil free products is based on the ingredient analyzer of Open Food Facts. For each product, we analyze the ingredients list to separate each individual ingredient and look for the corresponding ingredient in our ingredients taxonomy.</p>
+<p>Taxonomies are where we gather a lot of information about various things like ingredients, additives, food categories…</p>
+<p>The ingredients taxonomy is a huge multilingual database of ingredients with their possible synonyms. We are constantly expanding the ingredients taxonomy in order to be able to recognize more and more ingredients in more and more languages.</p>
+<p>The support of synonyms and hierarchy in ingredients means that the parser will be handle ingredients that are often confusing to the novice ingredient list reader.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/ingredients-analysis.html:31]:
```suggestion
<p>The support of synonyms and hierarchy in ingredients means that the parser can handle ingredients that are often confusing to the novice ingredient list reader.</p>
```

Not sure about the "novice ingredient list reader", that could be perceived as condescending.

File: lang/en/texts/ingredients-analysis.html (hunk 3 above)
Comment (teolemon) [inline lang/en/texts/ingredients-analysis.html:26]:
```suggestion
```

Comment (teolemon) [reply lang/en/texts/ingredients-analysis.html:26]: @stephanegigandet removing this ?

File: lang/en/texts/ingredients-analysis.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+<p>We hope that this feature will save you time, either on your path to adopting a flexitarian, vegetarian or vegan diet, or for other reasons.</p>
+
+<p>The egg symbolizes the vegetarian diet, which allows animal derived products like milk, eggs and honey.</p>
+
+<p>The leaf symbolizes the vegan diet which only allows ingredients from vegetal sources.</p>
+
+<p>The monkey symbolizes the palm oil status, since it has been largely impacted by palm-oil related deforestation.</p>
+
+How it works
```
Comment (teolemon) [inline lang/en/texts/ingredients-analysis.html:22]:
```suggestion
<h2>How it works</h2>
```

**Pull request merged**